src/pnpq/units.py

```diff
@@ -2,5 +2,33 @@
 
 ureg = pint.UnitRegistry()
 
-# Custom unit definitions
-ureg.define("mpc320_step = (170 / 1370) degree")  # MPC320
+# Initialize the UnitRegistry
+ureg = pint.UnitRegistry()
+
+# Custom unit definitions for MPC320
+ureg.define("mpc320_step = (170 / 1370) degree")
+ureg.define("mpc320_max_velocity = 400 degree / second")
```

Comment:
Yeah, that is true. I thought it would be nice, because then it becomes obvious that the velocity is defined proportionally, but probably don't need to use this anywhere else, so it's kind of extraneous. I'll just make it a variable instead.